src/stores/options.json

```diff
@@ -92,11 +92,37 @@
         "itemType": "section",
         "label": "Fenix"
       },
+      {
+        "label": "Any Fenix or Firefox Preview Activity",
+        "key": "Any Preview+Fenix Activity",
+        "disabledDimensions": ["os", "attributed"],
+        "shortDescription": "The profile has sent a Glean 'baseline' ping from Fenix or Firefox Preview on the day in question",
+        "channels": [
+          { "label": "Release", "key": "release" },
+          { "label": "Beta", "key": "beta" },
+          { "label": "Nightly", "key": "nightly" }
+        ]
+      },
       {
         "label": "Any Fenix Activity",
         "key": "Any Fenix Activity",
-        "disabledDimensions": ["os", "channel", "attributed"],
-        "shortDescription": "The profile has sent a Glean 'baseline' or 'metrics' ping from Fenix on the day in question"
+        "disabledDimensions": ["os", "attributed"],
+        "shortDescription": "The profile has sent a Glean 'baseline' ping from Fenix on the day in question; excludes Firefox Preview",
+        "channels": [
+          { "label": "Release (org-mozilla-firefox)", "key": "release" },
+          { "label": "Beta (org-mozilla-firefox-beta)", "key": "beta" },
+          { "label": "Nightly (org-mozilla-fennec-aurora)", "key": "nightly" }
```

Comment:
Is is too noisy to add the specific application IDs along with the channel labels? I could instead move this information to the shortDescription. I think we need to make it readily available in the UI since the questions of "what does this channel label actually mean" keeps coming up for Fenix.

Reply:
I would add each application ID as `shortDescription` instead like the language explanations

Reply:
definitely worth keeping this in here though, to avoid any confusion